recipes/configs/qwen2/knowledge_distillation_single_device.yaml

```diff
@@ -44,9 +44,9 @@ checkpointer:
 
 teacher_checkpointer:
   _component_: torchtune.training.FullModelHFCheckpointer
-  checkpoint_dir: /tmp/Qwen2-1.5B-Instruct-lora-finetune
+  checkpoint_dir: /tmp/Qwen2-1.5B-Instruct
   checkpoint_files: [
-    hf_model_0001_0.pt
+    model.safetensors
   ]
   recipe_checkpoint: null
   output_dir: /tmp/Qwen2-1.5B-Instruct-lora-finetune
```

Comment:
should change output dir here

Reply:
i thought about it. Indeed in all of our configs we use the same output dir. I will make the change